providers/ytt/02_addons/management-packages/featuregates/add_default_featuregate_cr.yaml

```diff
@@ -1,43 +0,0 @@
-#@ load("@ytt:data", "data")
-#@ load("@ytt:yaml", "yaml")
-
-#@ def featuregates_default_resource():
-apiVersion: config.tanzu.vmware.com/v1alpha1
-kind: FeatureGate
-metadata:
-  name: tkg-system
-  annotations:
-    tkg.tanzu.vmware.com/addon-type: "packages/management-package"
-spec:
-  namespaceSelector: {}
-#@ end
-
-#@ if data.values.TKG_CLUSTER_ROLE != "workload" and data.values.PROVIDER_TYPE != "tkg-service-vsphere":
```

Comment:
question: wondering who is taking care of this logic now ?

Reply:
packages in the management package repository are exclusively installed on the management cluster, so we can ignore the first condition to check for workload cluster, and regarding TKGs they make the management packages available individually with a Package CR in the cluster, so we don't need to take care of that condition as well